manifests/child.pp

```diff
@@ -4,22 +4,18 @@
 # Install and configure Pulp node
 #
 class pulp::child (
-  $parent_fqdn           = undef,
-  $parent_qpid_scheme    = 'ssl',
-  $parent_qpid_port      = '5671',
-  $oauth_effective_user  = 'admin',
-  $oauth_key             = 'key',
-  $oauth_secret          = 'secret',
-  $ssl_cert              = '/etc/pki/pulp/ssl_apache.crt',
-  $ssl_key               = '/etc/pki/pulp/ssl_apache.key',
-  $server_ca_cert        = '/etc/pki/pulp/ca.crt'
+  String[1] $parent_fqdn,
+  String $oauth_effective_user = 'admin',
```

Comment:
It may be, I think pulp::child is intended to be used standalone so it should use it's own values.